src/ray/protobuf/gcs.proto

```diff
@@ -31,21 +31,22 @@ enum TablePrefix {
   NODE = 4;
   OBJECT = 5;
   ACTOR = 6;
-  FUNCTION = 7;
-  TASK_RECONSTRUCTION = 8;
-  RESOURCE_USAGE_BATCH = 9;
-  JOB = 10;
-  PROFILE = 11;
-  TASK_LEASE = 12;
-  NODE_RESOURCE = 13;
-  DIRECT_ACTOR = 14;
+  ACTOR_TASK_SPEC = 7;
```

Comment:
Can you just keep the prefix ordering and add `ACTOR_TASK_SPEC =21`?

Reply:
No problem. Putting it after `ACTOR` makes more readability as they are related. I'd like adding it just in the end. I don't have a strong preference here.